CHANGELOG.md

```diff
@@ -184,14 +185,18 @@ arguments.
   This issue could cause enum table values to sometimes not be properly reloaded without restarting `graphql-engine`. Now a `reload_metadata` API call (or clicking “Reload enum values” in the console) should consistently force a reload of all enum table values.
 - server: fix event trigger cleanup on deletion via replace_metadata (fix #5461) (#6137)
 **WARNING**: This can cause significant load on PG on startup if you have lots of event triggers. Delay in starting up is expected.
+- console: allow user to cascade Postgres dependencies when dropping Postgres objects (close #5109) (#5248)
+- console: mark inconsistent remote schemas in the UI (close #5093) (#5181)
+- console: remove ONLY as default for ALTER TABLE in column alter operations (close #5512) #5706
+- console: add option to flag an insertion as a migration from `Data` section (close #1766) (#4933)
 - console: add notifications (#5070)
 - cli: fix bug in metadata apply which made the server aquire some redundant and unnecessary locks (close #6115)
 - cli: fix cli-migrations-v2 image failing to run as a non root user (close #4651, close #5333)
 - cli: fix issue with cli binary on latest Mac (Big Sur) (fix #5462)
 - docs: add docs page on networking with docker (close #4346) (#4811)
 - docs: add tabs for console / cli / api workflows (close #3593) (#4948)
 - docs: add postgres concepts page to docs (close #4440) (#4471)
-- docs: add guides on connecting hasura cloud to pg databases of different cloud vendors (#5948)
+- docs: add guides on connecting hasura cloud to pg databases of different cloud vendors (#5948) 
```

Comment:
Where did this come from? I don't see it elsewhere in the changelog

Reply:
Really not sure about this. Removing it.